### PR TITLE
Cldc 1521 living before purchase

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**
image

CYA page - should this be "year" when `proplen` = 1 or is it fine to always be "years" as shown:
image

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -415,6 +415,14 @@
...
     t.integer "ecstat5"
     t.integer "ecstat6"
+    t.integer "pregyrha"
+    t.integer "pregla"
+    t.integer "pregghb"
+    t.integer "pregother"
+    t.integer "disabled"
+    t.integer "wheel_value_check"
+    t.integer "hhregres"
+    t.integer "hhregresstill"
```
Comment (natdeanlewissoftwire) [inline db/schema.rb]: should these be included?

Comment (kosiakkatrina) [reply db/schema.rb]: I don't think so, try dropping and recreating the db, it should update it

Comment (natdeanlewissoftwire) [reply db/schema.rb]: Cheers!

**Review by kosiakkatrina — COMMENTED** [review]
Looks good, just needs a rebase!

**Pull request merged**